Resources/Prototypes/Objectives/objectiveGroups.yml

```diff
@@ -31,7 +31,7 @@
 - type: weightedRandom
   id: TraitorObjectiveGroupKill
   weights:
-    KillRandomPersonObjective: 1
+    # KillRandomPersonObjective: 1 Floofstation Edit
     KillRandomHeadObjective: 0.25
```

Comment:
This does not remove kill objectives, just removes the chance for a kill objective to be a *regular kill objective*. This means that heads will become 5 times as likely to be chosen as kill targets than before.

Reply:
Yeah.